liquidator/index.js

```diff
@@ -61,33 +62,39 @@ async function run(
 
     const getTime = () => Math.round(new Date().getTime() / 1000);
 
-    // Setup web3 accounts, account instance and pricefeed for EMP.
-    const [accounts, emp, voting, priceFeed] = await Promise.all([
+    // Load unlocked web3 accounts, get the networkId and set up price feed.
+    const [accounts, networkId, priceFeed] = await Promise.all([
       web3.eth.getAccounts(),
-      ExpiringMultiParty.at(empAddress),
-      Voting.deployed(),
+      web3.eth.net.getId(),
       createReferencePriceFeedForEmp(logger, web3, new Networker(logger), getTime, empAddress, priceFeedConfig)
     ]);
 
     if (!priceFeed) {
       throw new Error("Price feed config is invalid");
     }
 
+    // Setup contract instances.
+    const voting = new web3.eth.Contract(getAbi("Voting"), getAddress("Voting", networkId));
+    const emp = new web3.eth.Contract(getAbi("ExpiringMultiParty"), empAddress);
```

Comment:
Does this work in tests? If so, how?

Reply:
`getAddress` returnes undefined :) 

This means that the voting instance does not actually exist and a `null` type is passed to the `Liquidator`. This doesn't mess with any of the tests though as this `voting` instance is literally only used to inform [logs](https://github.com/UMAprotocol/protocol/blob/dccb2feb9e5aa883648ee7dab4896c1bc6d92694/liquidator/liquidator.js#L420) and this functionality is not tested in the `index.js` tests at all.

Reply:
I see. This is kinda weird and may not be a pattern we can follow elsewhere. Thoughts on adding a TODO to make this correctly resolve in tests?

Reply:
I've added a note on this in the relevant part of code.